packages/@aws-cdk/aws-stepfunctions/lib/states/task.ts

```diff
@@ -34,6 +34,13 @@ export interface TaskProps {
      */
     inputPath?: string;
 
+    /**
+     * Parameters pass a collection of key-value pairs, either static values or JSONPath expressions that select from the input.
```

Comment:
Might be interesting to add a link to the StepFunctions doc about how to encode the key-value pairs, although since the other fields don't have those documentation pointers, maybe that'd stand out too much.

Reply:
I added the link as there is no harm in having it there :)